.github/workflows/triviy.yml

```diff
@@ -0,0 +1,84 @@
+name: Trivy scan
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '46 8 * * 0'
```

Comment:
Why a schedule here instead of just on landing?  I.e.:
```suggestion
  push:		
    branches: ["master"]		
```

This way we know exactly which landing may have caused a regression and don't have to bisect 24h worth landings to find the one that is causing a failure.

Reply:
There are two factors in Trivy scans:
- a new code that enables new use cases and new potential vulnerability areas
- new vulnerabilities detected in already used packages

I assume that the second bullet will be the root cause of the majority of incidents

Reply:
If I'm understanding the first bullet, then that is something that the developer introduced with their PR and so they need to resolve it there.

Even the second bullet does not represent cases we don't already experience.  Any one of our linters for example can start flagging pre-existing code that it never used to simply because a new version of the linter is introduced and starts flagging existing constructs that it does not like any more.  When this happens we have to fix the code it starts flagging.  It fails PRs.  Because if it doesn't it is ignored and landed and then everyone's PRs start flagging the offending (yet pre-existing) code.

Reply:
I understand the risk and we already accept that kind of risk and resolution process with our other checkers.  We frequently see unrelated linting issues creep into PRs due to updated linting tools, spelling checkers, etc.

Those are either corrected in the PR that found them or somebody is assigned the task of getting master updated to the correction.  But it's only because they block progress that they get fixed.  Otherwise nobody takes on the task and distraction to address them if they don't halt progress.  History has proven this to us.

PRs that are blocked wait for the update and rebase, or they _can_ be force-landed if gatekeepers are willing.

Given that we accept this process for less serious issues like spell-checking and code linting, we must surely accept this process for actual security vulnerabilities that are detected!